src/components/UserDetailsTooltip/index.js

```diff
@@ -28,11 +28,11 @@ function UserDetailsTooltip(props) {
                 </Text>
 
                 <Text style={[styles.textMicro, styles.fontColorReactionLabel]}>
-                    {String(userDetails.login).trim() && !_.isEqual(userDetails.login, userDetails.displayName) ? Str.removeSMSDomain(userDetails.login) : ''}
+                    {String(userDetails.login || '').trim() && !_.isEqual(userDetails.login, userDetails.displayName) ? Str.removeSMSDomain(userDetails.login) : ''}
```

Comment:
Can we add `''` as a default prop to `userDetailsTooltipPropTypes` instead?

Reply:
I don't think so, because in [this web-e PR](https://github.com/Expensify/Web-Expensify/pull/37753/files) we're going to completely stop sending `login` (& other details) for public users. But the personal details WILL include avatar, accountID, displayName... So I don't believe we'll insert any default data for those situations